model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -180,6 +180,7 @@
             "mistral-7b-instruct",
             "mixtral-8x7b",
             "mixtral-8x7b-instruct",
+            "mixtral-8x22b",
```

Comment:
is there an instruct version to add too?

Reply:
don't see an instruct version from huggingface